codex-rs/core/src/codex.rs

```diff
@@ -1164,6 +1168,14 @@ async fn submission_loop(
             Op::Interrupt => {
                 sess.interrupt_task().await;
             }
+            Op::UndoLastSnapshot => {
```

Comment:
What is the UX around this? Pet the previous comment, I would expect snapshots and undo to have some sort of identifier so you "undo to X", for example.